csrc/xpu/cutlass_kernels/grouped_gemm.hpp

```diff
@@ -8,15 +8,26 @@
 /* #include "pytorch_shim.h" */
 
 #include <torch/all.h>
+
+#include "collective/gemm/moe_dtype_policy.hpp"
 #include "utils.h"
 
 namespace gpu::cutlass_kernel {
 
 namespace grouped_gemm {
+template <class moe_policy>
```

Comment:
We can do a template like this:
```
template<typename ElementType>
void kernel_functor

...

kernel_functor<cutlass::half_t>(args..)
```
ref:https://github.com/intel/sycl-tla/blob/main/examples/09_bmg_grouped_gemm_f8/09_bmg_grouped_gemm_f8.cpp#L559

Reply:
You're right.